server/routes/assessmentsRoute.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+
+const assessmentsRouter = express.Router()
+
+router.get('/assessments/:id', (req, res) => {
+  const id = req.params.id
+})
+
+router.put('/assessments/:id', (req, res) => {
+  const assID = req.params.id
```

Comment:
Can we change this variable name? 😂

We are in the assessments routes so I don't think prefixing it is necessary, and it would be inconsistent with the variable name on line 6.

Generally the shorthand is `id`, not `ID` as it not an acronym, and it can be difficult to read (e.g. https://stackoverflow.com/a/13862674).